src/config.js

```diff
@@ -93,6 +93,10 @@ export default class Config {
 
   save (key, value) {
     this.config[key] = value
+    if (key == 'creds') {
+      // Persist the access token after it has been refreshed
+      this.persist()
+    }
```

Comment:
I'm not very familiar with the auth process yet. But from what I understand, a key `'foo'` could be added to `config` but not persisted, then `'creds'` would be added and both of them persisted...
Then why don't we just persist on every `save()` call?

(Also eslint doesn't like the `==` comparison)